src/pages/Homepage.js

```diff
@@ -47,11 +47,13 @@ const Homepage = (props) => {
     <div>
       <h1>Task Master</h1>
       <Form addTodo={addTask} />
-      <TodoList
-        refreshTodos={refreshTasks}
-        deleteTodo={deleteTask}
-        todos={tasks}
-      />
+      {tasks.length > 0
```

Comment:
Do we know `tasks` will always be an array? This will fail if `tasks === null` for example.